lib/intercom_flutter.dart

```diff
@@ -100,6 +100,10 @@ class Intercom {
     });
   }
 
```

Comment:
Could you add a doc comment here explaining what does this method do? Something like:
```dart
/// Sets Intercom's chat `padding` (not really sure).
```

Reply:
Thank you for review. I implemented these wanted changes to new commit. I still cannot be sure that iOS implementation is correct because I have not tested (or compiled) these on iOS platform.